src/libANGLE/renderer/Format_table_autogen.cpp

```diff
@@ -18,7 +18,7 @@ namespace angle
 {
 
 static constexpr rx::FastCopyFunctionMap::Entry BGRAEntry  = {angle::FormatID::R8G8B8A8_UNORM,
-                                                             CopyBGRA8ToRGBA8};
+                                                              CopyBGRA8ToRGBA8};
```

Comment:
Where do these formatting changes come from? Visual Studio Code? I think it is better to keep the diff as small as possible and revert these

Reply:
These autogen files are auto-formatted by the script, previous committed state didn't adhere to it probably accidentally.

Reply:
It's somehow done by the generator, but I reverted in: https://github.com/shapr3d/angle/pull/20/commits/0076be8a45a6a92300452e8580f33f074fdd53a3